core/logger/Logger.cpp

```diff
@@ -267,14 +286,32 @@ void Logger::LoadConfig(const std::string& filePath) {
         logConfigInfo = "Load log config from " + filePath;
     } while (0);
 
+    // parse env log level
+    level::level_enum* envLogLevel = new(level::level_enum);
+    std::string aliyun_logtail_log_level = "";
+    std::string logLevelInfo;
+    if (ReadLogLevelFromEnv(envLogLevel, aliyun_logtail_log_level)) {
+        if (!aliyun_logtail_log_level.empty()) {
+            logLevelInfo = "read log level from the env successfully, level: " + aliyun_logtail_log_level;
+        }
+        for (auto& loggerConfig : loggerConfigs) {
+            loggerConfig.second.level = *envLogLevel;
+        }
+    } else {
+        logLevelInfo = "read log level from the env error, level: " + aliyun_logtail_log_level;
+    }
+    if (!logLevelInfo.empty()) {
+        LogMsg(logLevelInfo);
```

Comment:
maybe we can only print a log when read log level successed to simplify the code
and set aliyun_logtail_log_level="" if ReadLogLevelFromEnv returns false
If you think a warning is appropriate here, you can do this in the else branch. however I don't think the logLevelInfo variable is necessary.

Reply:
Yes